ARM/README.md

```diff
@@ -0,0 +1,88 @@
+##Configuring Datadog in ARM Templates
+
+ - [Parameters to accept Datadog configuration. ](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/parameters)
+  - DD_API_KEY
+  - DD_ENV
+  - DD_SERVICE
+  - DD_VERSION
+  - DD_SITE
+ - [Entries in `appSettings` underneath `Microsoft.Web/sites/config`](https://docs.microsoft.com/en-us/azure/templates/microsoft.web/sites/config-web?tabs=json)
+	 - One to one entries for each relevant parameter
+
+*To enable features such as [Runtime Metrics](https://docs.datadoghq.com/tracing/runtime_metrics/dotnet/) or [Logs Correlation](https://docs.datadoghq.com/tracing/connect_logs_and_traces/dotnet/?tab=serilog), you can specify these in your `appSettings` section, or you can pass them as parameters similar to the other values specified in the example below.*
+
+```
+{
+    ...
+    "parameters": {
+        ...
+        "dd_api_key": {
+            "defaultValue": "missing-dd-api-key-parameter",
```

Comment:
I don't think we should supply default values for api key, env, service, version. Using the default values would be incorrect, so I think it makes more sense for users that they error if they don't supply them

Reply:
Agreed.